examples/hosts/app-integration/external-controller/package.json

```diff
@@ -49,6 +49,7 @@
     "style-loader": "^1.0.0"
   },
   "devDependencies": {
+    "@fluid-experimental/get-container": "^0.39.0",
```

Comment:
Why?

Reply:
The tests here still use `getSessionStorageContainer` from `get-container`